string type
string data
---
bool success